src/DaPigGuy/PiggyFactions/info/FactionInfo.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DaPigGuy\PiggyFactions\info;
+
+use function count;
+use DaPigGuy\PiggyFactions\factions\Faction;
+use DaPigGuy\PiggyFactions\players\PlayerManager;
+use SOFe\InfoAPI\{Info, InfoAPI, NumberInfo, PlayerInfo, StringInfo};
+use pocketmine\Server;
+
+final class FactionInfo extends Info
+{
+    /** @var Faction */
+    private $faction;
+
+    public function __construct(Faction $faction) {
+        $this->faction = $faction;
+    }
+
+    public function toString() : string {
+        return $this->faction->getName();
+    }
+
+    public function getInfoType() : string {
+        return "faction";
+    }
+
+    public static function register() : void {
+        InfoAPI::provideInfo(self::class, StringInfo::class, "piggyfactions.faction.name", fn($info) => new StringInfo($info->faction->getName()));
+        InfoAPI::provideInfo(self::class, StringInfo::class, "piggyfactions.faction.description", fn($info) => new StringInfo($info->faction->getDescription()));
+        InfoAPI::provideInfo(self::class, StringInfo::class, "piggyfactions.faction.motd", fn($info) => new StringInfo($info->faction->getMotd()));
+        InfoAPI::provideInfo(self::class, NumberInfo::class, "piggyfactions.faction.power", fn($info) => new NumberInfo($info->faction->getPower()));
+        InfoAPI::provideInfo(self::class, NumberInfo::class, "piggyfactions.faction.maxPower", fn($info) => new NumberInfo($info->faction->getMaxPower()));
```

Comment:
Consider merging `power` and `maxPower` into a RatioInfo: https://github.com/SOF3/InfoAPI/blob/7d266afd911aaf64432f1485b7237404729b05f9/src/SOFe/InfoAPI/PlayerInfo.php#L80-L84